addons/sms/tests/test_sms_template.py

```diff
@@ -106,3 +108,28 @@ def test_sms_template_rendering_unrestricted(self):
 
         body = sms_composer._prepare_body_values(self.partner)[self.partner.id]
         self.assertIn(self.partner.name, body, 'Template Editor should be able to write new Jinja code')
+
+
+@tagged('post_install')
+class TestSmsTemplateReset(TransactionCase):
+
+    def _load(self, module, *args):
+        convert_file(self.cr, module='sms',
+                     filename=get_module_resource(module, *args),
+                     idref={}, mode='init', noupdate=False, kind='test')
+
+    def test_sms_template_reset(self):
```

Comment:
Note that we could also add a unit test with the wizard so that its code is covered